modules/serverless-workflow-basic.adoc

```diff
@@ -0,0 +1,125 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="basic-workflow_{context}"]
+= Basic workflow using Knative Client
+
+Use this basic workflow to create, read, update, delete (CRUD) operations on a service. The following example deploys a link:https://github.com/knative/docs/tree/master/docs/serving/samples/hello-world/helloworld-go[simple Hello World service] that reads the environment variable `TARGET` and prints its output.
+
+* **Create a service in the `default` namespace from an image**
++
+[source]
+----
+kn service create hello --image gcr.io/knative-samples/helloworld-go --env TARGET=Knative
```

Comment:
Needs a $ in front of these commands. Double check - there are a few other places in the PR missing them too.